js/github.js

```diff
@@ -0,0 +1,40 @@
+// Helpers for the GitHub API.
+
+define(
+    [],
+    function () {
+        function findNext(header) {
+            var m = (header||"").match(/<([^>]+)>\s*;\s*rel="next"/);
+            return (m && m[1]) || null;
+        }
+        
+        function fetch(url) {
+            return $.ajax(url);
```

Comment:
The return promise isn't fulfilled with the same thing (JSON here, a `Request` object for `fetch`).